### DISCUSSION TITLE
모임 참가 신청

### DISCUSSION THREAD

**Discussion by seriousran** [Pseudo-Lab/growth-collector#3] [Announcements]
[모임 참가 신청 작성 방법]
- 모임 공지 아래에 댓글로 양식에 맞추어 참가 신청하시면 됩니다.
- 신청하시고, 별다른 승인 절차 없이 모임 장소에 오시면 됩니다!
- 양식: 이름 / n번째 참여 / 이메일 주소 (구글 캘린더용; 1번째 참여시에만 작성) / 모임을 기대하며 한마디
  - ex) 김찬란 / N번째 참여 / seriousran@gmail.com / 2주에 한번이지만, 의미있는 시간이 되길 기대합니다!

**Comment by seriousran** [comment]
[7월 25일 모임 공지]

TL;DR
- 이 서클은 성장을 수집하고 나누는 방식으로 운영합니다! 국내외 AI 분야의 전문가들을 찾고, 그들의 성장 과정을 살펴보며, 그들의 이야기를 주제 삼아 대화합니다. (혹시 진행 및 정리가 잘 된다면, 인명사전도 만들면 좋을 것 같습니다.)
- 처음 오신 분들, 가짜연에 참여하고 싶지만 어디로 갈지 모르시겠다면, 저랑 시작하시면 됩니다!
- 빌더 분들 및 각종 전문가들을 초대해 성장 이야기를 공유하는 것 까지 목표로 합니다!

모임은 월 2회 진행하며, 오프라인 위주로 진행하고자 합니다!

7/25(목) 저녁 7시 광화문역에서 첫 모임을 가질 예정이며,
주로 광화문/종각/을지로 일대에서 모임을 가지려합니다.

첫 모임이기에 자기소개, 자신의 하반기 목표 등을 공유하는 시간을 갖고,
관심있는 국내외 AI 분야 전문가에 대해서 가볍게 이야기 나누고자 합니다!
가벼운 마음으로 오세요! :)

**Reply by seriousran** [reply]
7월 25일 / 김찬란 / 1번째 참여 / [seriousran@gmail.com](mailto:seriousran@gmail.com)

**Reply by shinhyunji36** [reply]
7월 25일 / 신현지 / 1번째 참여 / lej70445@gmail.com

**Reply by soncy0311** [reply]
7월 25일 / 손창엽 / 1번째 참여 / thsckdduq@gmail.com

**Reply by yunhee1** [reply]
7월 25일 / 황윤희 / 1번째 참여 / alkrg001@gmail.com

**Reply by ZeVicTech** [reply]
7월 25일 / 김영승 / 1번째 참여 / kys980501@gmail.com

**Reply by seriousran** [reply]
@shinhyunji36 @soncy0311 @yunhee1 @ZeVicTech 
모임 신청해주신 분들께 감사드려요! 재밌는 이야기 함께 나눠요! 광화문역 7번출구 앞에서 뵙겠습니다!! 혹시 좀 늦거나 엇갈린경우 010-8949-1771로 연락주세요!

**Reply by shinhyunji36** [reply]
안녕하세요, 9번 출구가 아니라 7번 출구에서 뵙는게 맞을까요?! 디스코드에서는 9번 출구라고 적어주신 걸 봐서요!☺️

**Reply by soncy0311** [reply]
저는 7번출구로 나왔다가 너무 더워서 올리브영으로 피신했네요..😭

**Comment by seriousran** [comment]
[8월 8일 모임 공지]

TL;DR
두번째 모임이지만 여전히 서로를 알아가기 위해 OKR 수립 구체화와 액션 아이템들을 함께 생각해보고- 2주간의 시행착오와 경험들을 공유하고자 합니다! 또한, 관심있는 국내외 AI 분야 전문가에 대해서 가볍게 이야기 나누고자 합니다!

8/8(목) 저녁 7시 스타벅스 을지로국제빌딩점에서 모임을 가질 예정이며, 주로 광화문/종각/을지로 일대에서 모임을 가지려합니다.

열린 마음으로 오세요! :)

**Reply by shinhyunji36** [reply]
8월 8일 / 신현지 / 2번째 참여 / [lej70445@gmail.com](mailto:lej70445@gmail.com)

**Reply by soncy0311** [reply]
8월8일 / 손창엽 / 2번째 참여

**Reply by ZeVicTech** [reply]
8월 8일 / 김영승 / 2번째 참여

**Reply by yesinkim** [reply]
8월 8일 / 김예신 / 1번째 참여 / bailando.ys@gmail.com
(9시에 다른 일정이 있어 먼저 일어날 수 있는 점 미리 공유드립니다 :))

**Reply by seriousran** [reply]
@yesinkim 님! 아래의 주소로 변경되었습니다! 혹시 저희를 못찾으실 경우 01089491771로 연락주세요!

[네이버 지도]
투썸플레이스 을지로입구역점
서울 중구 남대문로 92
https://naver.me/5xnpf1y8

**Reply by yesinkim** [reply]
@seriousran 일찍 도착해서 같이 이동했습니다! 곧 뵙겠습니다🤩

**Comment by seriousran** [comment]
[8월 22일 모임 공지]

TL;DR
지난번에 서로의 교차점도 알아보며, 이해도를 높였습니다! 이제는 OKR 달성을 위한 액션 아이템들을 함께 진행 및 점검해보는 시간을 갖습니다! 그리고 "나의 성장을 가로막는 벽"에 대해 이야기하며, 변화된 나의 모습도 기대해보는 시간을 가지려합니다. 마지막으로, 관심있는 국내외 AI 분야 전문가에 대해서 가볍게 이야기 나누고자 합니다!

8/22(목) 저녁 7시 투썸플레이스 을지로입구역점에서 모임을 가질 예정입니다!
(저희는 주로 광화문/종각/을지로 일대에서 모임을 가집니다!)

열린 마음으로 오세요! :)
*너무 늦은 공지 죄송합니다...

**Reply by seriousran** [reply]
김찬란 / 3번째 참여! / 숙제는 아직 못했지만... 내일까지 조금이라도 더 해보겠습니다! :)

**Reply by soncy0311** [reply]
손창엽 / 3번째 참여 / 저도 숙제를 못했네요.... 이번 2주동안은 밀린 숙제를 꼭 해내보겠습니다...!

**Reply by yunhee1** [reply]
황윤희 / 2번째 참여 / 지난 번에 참석을 못해 따라잡는 자리가 될 것 같습니다..!

**Reply by leh60245** [reply]
이의형 / 1번째 참여 / kkh60245@gmail.com / 가벼운 마음으로 처음 참여하고자 합니다!

**Reply by ZeVicTech** [reply]
김영승 / 3번째 참여 / 저도 숙제를 다 하지는 못했지만.. 할 수 있는 데까지 해서 참석하겠습니다!

**Reply by seriousran** [reply]
@leh60245 이의형님! 혹시 도착하시면 01089491771로 연락주세요!

**Comment by seriousran** [comment]
**[9월 5일 모임 공지]**

TL;DR
각자의 OKR 달성을 위한 액션 아이템들을 함께 진행 및 점검해보는 시간을 갖습니다! 그리고 9기 가짜연구소 활동에 대해 간단히 이야기를 나누고, 마지막으로, 관심있는 국내외 AI 분야 전문가에 대해서 가볍게 이야기 나누고자 합니다!

9/5(목) 저녁 8시반 온라인에서 모임을 가질 예정입니다!
[가짜연구소 디스코드](https://discord.gg/EPurkHVtp2) > 아카데미 > #Room-GH 에서 진행합니다.
(저희는 주로 광화문/종각/을지로 일대에서 모임을 가집니다!)

열린 마음으로 오세요! :)

**Reply by yunhee1** [reply]
황윤희 / 3번째 참여 / 온라인에서 뵙겠습니다 :)

**Reply by ZeVicTech** [reply]
김영승 / 4번째 참여 / 저녁에 뵙겠습니다!

**Comment by seriousran** [comment]
[9월 19일 모임 공지]

TL;DR
각자의 OKR 달성을 위한 액션 아이템들을 함께 진행 및 점검해보는 시간을 갖습니다! 그리고 마법학교 <성장 수집가>의 컨셉에 대해 디자인해보고, 관심있는 국내외 AI 분야 전문가에 대해서 가볍게 이야기 나누고자 합니다!

9/19(목) 저녁 7시 투썸플레이스 을지로입구역점에서 모임을 가질 예정입니다!
[가짜연구소 디스코드](https://discord.gg/EPurkHVtp2) > 아카데미 > #Room-GH 에서 진행합니다.
(저희는 주로 광화문/종각/을지로 일대에서 모임을 가집니다!)

열린 마음으로 오세요! :)

**Reply by soncy0311** [reply]
손창엽 / 4번째 참여 / 퇴근하고 가면 7시반쯤 될것 같습니다! 최대한 빨리 가겠습니다🫠

**Reply by yunhee1** [reply]
황윤희 / 4번째 참여 / 새 글로 올려서.. ㅎㅎㅎ 댓글로 다시 작성합니다!

**Comment by yunhee1** [comment]
황윤희 / 4번째 참여 / 내일 뵙겠습니다!

**Comment by seriousran** [comment]
[10월 17일 모임 공지]

TL;DR

성장에 대한 이야기를 나누고, 성장하기 위한 가설을 세우고 실험을 진행합니다. 각자의 OKR 달성을 위한 액션 아이템들을 함께 진행 및 점검해보는 시간을 갖습니다! 그리고 마법학교 <성장 수집가>의 규칙을 구체화하고, 관심있는 국내외 AI 분야 전문가에 대해서 가볍게 이야기 나누고자 합니다!

10/17(목) 저녁 7시 투썸플레이스 을지로입구역점에서 모임을 가질 예정입니다!
(저희는 주로 광화문/종각/을지로 일대에서 모임을 가집니다!)

열린 마음으로 오세요! :)

**Reply by yunhee1** [reply]
황윤희 / 6번째 참여 / 내일 뵙겠습니다💪

**Reply by soncy0311** [reply]
손창엽 / 5번째 참여 / 퇴근이 좀 늦을 수도 있을 것 같아서...최대한 빨리 가겠습니다!

**Reply by ZeVicTech** [reply]
김영승 / 6번째 참여 / 9시에 일정이 있어서 8시반까지만 참여가능할 것 같습니다..ㅠ

**Comment by seriousran** [comment]
[10월 31일 모임 공지]

TL;DR

각자의 성장에 대한 이야기를 나누고, 성장하기 위한 가설을 세우며 실험을 진행합니다. 각자의 OKR/목표 달성을 위한 액션 아이템들을 점검해보며, 서로의 성장의 과정들을 나누는 시간도 갖습니다. 요즘 AI 업계에서 나눌 흥미로운 주제들로 이야기도 나눠봅니다.

특히 이번 모임은 할로윈 데이에 진행되는 만큼, 할로윈 데이 파티까지는 아니지만...- 간단히 기념 사진 정도 찍어볼까 합니다! 재미난 소품이 있다면 가져와서 함께 사진찍어요!

10/31(목) 저녁 7시 투썸플레이스 서린동청계광장점 지하1층 Room1에서 모임을 가질 예정입니다! (저희는 주로 광화문/종각/을지로 일대에서 모임을 가집니다!)

열린 마음으로 오세요! :)

**Reply by yunhee1** [reply]
황윤희 / 7번째 참여 / 할로윈에 성장수집가 분들을 만나다니 벌써 기대됩니다🎃

**Reply by soncy0311** [reply]
손창엽 / 6번째 참여 / 오늘도 약간 늦을것 같은데, 최대한 빨리 가겠습니다! 😢

**Comment by seriousran** [comment]
[11월 14일 모임 공지]

TL;DR

각자의 성장에 대한 이야기를 나누고, 성장하기 위한 가설을 세우며 실험을 진행합니다. 각자의 OKR/목표 달성을 위한 액션 아이템들을 점검해보며, 서로의 성장의 과정들을 나누는 시간도 갖습니다. 요즘 AI 업계에서 나눌 흥미로운 주제들로 이야기도 나눠봅니다.

특히 이번 모임은 11/23(토)에 진행될 가짜연구소의 행사에서 어떤 이야기를 나눌지 준비해보려 합니다!! :)

11/14(목) 저녁 7시 투썸플레이스 서린동청계광장점 지하1층 Room1에서 모임을 가질 예정입니다! (저희는 주로 광화문/종각/을지로 일대에서 모임을 가집니다!)

열린 마음으로 오세요! :)

- 댓글 작성 ex) 이름 / N번째 참여 / 기대하는 한마디!

**Reply by SunandBean** [reply]
~~김선빈 / 2번째 참여 / 시간 빠르네요! 조금씩이라도 성장해서 뵙겠습니다 ㅎㅎ~~
개인 사정으로 불참으로 변경하겠습니다.

**Reply by soncy0311** [reply]
손창엽 / 7번째 참여 / 목요일 전에 액션 아이템을 꼭 끝내고 가보겠습니다...!

**Reply by yunhee1** [reply]
황윤희 / 8번째 참여 / 가짜연구소 행사 너무 기대됩니다..!!!

**Comment by seriousran** [comment]
[11월 28일 모임 공지]

신청 링크: https://github.com/Pseudo-Lab/growth-collector/discussions/3

TL;DR

각자의 성장에 대한 이야기를 나누고, 성장하기 위한 가설을 세우며 실험을 진행합니다. 각자의 OKR/목표 달성을 위한 액션 아이템들을 점검해보며, 서로의 성장의 과정들을 나누는 시간도 갖습니다. 요즘 AI 업계에서 나눌 흥미로운 주제들로 이야기도 나눠봅니다.

특히 이번 모임은 11/23(토)에 진행됐던 가짜연구소의 행사에 대한 이야기도 나눠보려 합니다!! :)

11/28(목) 저녁 7시 투썸플레이스 서린동청계광장점 지하1층 Room1에서 모임을 가질 예정입니다! (저희는 주로 광화문/종각/을지로 일대에서 모임을 가집니다!)

열린 마음으로 오세요! :)

**Reply by SunandBean** [reply]
김선빈 / 2번째 참여 / 행사 후기도 기대되네요!

**Reply by yunhee1** [reply]
황윤희 / 9번째 참여 / 이번 행사에서 느낀 즐거움을 다른 분들과 나눌 수 있을 것 같아 기대됩니다!

**Reply by soncy0311** [reply]
손창엽 / 8번째 참여 / 이번주 액션 아이템을 못 세웠는데, 경제 이슈 정리랑 아비투스 읽는 것으로 대체하겠습니다!

**Comment by seriousran** [comment]
[12월 12일 모임 공지]

신청 링크: #3

TL;DR

각자의 성장에 대한 이야기를 나누고, 성장하기 위한 가설을 세우며 실험을 진행합니다. 각자의 OKR/목표 달성을 위한 액션 아이템들을 점검해보며, 서로의 성장의 과정들을 나누는 시간도 갖습니다. 요즘 AI 업계에서 나눌 흥미로운 주제들로 이야기도 나눠봅니다.

특히 이번 모임에서는 올해 "회고"에 대한 이야기를 해보려 합니다!

12/12(목) 저녁 7시 투썸플레이스 서린동청계광장점 지하1층 Room1에서 모임을 가질 예정입니다!

열린 마음으로 오세요! 😀 

*신규 참여자의 경우 기존 멤버의 초대를 받아 게스트로 참여할 수 있으며, 게스트로 3회 참여시 멤버로 편입할 수 있습니다!
*저희는 주로 광화문/종각/을지로 일대에서 모임을 가집니다!

**Reply by SunandBean** [reply]
김선빈 / 3번째 참여 / 회고 이야기 기대됩니다~!

**Reply by rlaalsrl0922** [reply]
김민기 / 2번째 참여 / 듣기만해도 성장할 수 있는 모임 너무 좋습니당,,~

**Reply by soncy0311** [reply]
손창엽 / 9번째 참여 /
